pytensor/compile/compilelock.py

```diff
@@ -35,7 +36,7 @@ def force_unlock(lock_dir: os.PathLike):
         Path to a directory that was locked with `lock_ctx`.
     """
 
-    fl = filelock.FileLock(os.path.join(lock_dir, ".lock"))
+    fl = filelock.FileLock(str(Path(lock_dir) / ".lock"))
```

Comment:
Thanks, I removed it and updated `filelock` to a newer version that allows that, as well as removed `types-filelock` which has been included in `filelock` itself since then (and was causing problems with `mypy` otherwise).